docker.local/bin/build.benchmark.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+cmd="build"
+
+docker $cmd -f docker.local/build.benchmarks/Dockerfile . -t zchain_benchmarks
+
+if [ "$#" -ne 3 ]; 
+    then echo "illegal number of parameters usage: docker.local/bin/gen_keys.sh signatureScheme [bls0chain or ed25519] absolute_path_to_keyfiles_folder key_file_name"
```

Comment:
Rather than return an error can we use sensible defaults? Otherwise, everyone will need to study this file before they can use it.